pktoken/pktoken_test.go

```diff
@@ -0,0 +1,60 @@
+package pktoken_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/openpubkey/openpubkey/pktoken"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPkToken(t *testing.T) {
+	TestPKTokenJSON(t)
+}
+
+func TestPKTokenJSON(t *testing.T) {
+	mockPkToken := map[string]any{
+		"payload": "eyJpc3MiOiJodHRwczovL2FjY291bnRzLmdvb2dsZS5jb20iLCJhenAiOiIxODQ5NjgxMzg5MzgtZzFmZGRsNXRnbG83bW5sYmRhazhoYnNxaGhmNzlmMzIuYXBwcy5nb29nbGV1c2VyY29udGVudC5jb20iLCJhdWQiOiIxODQ5NjgxMzg5MzgtZzFmZGRsNXRnbG83bW5sYmRhazhoYnNxaGhmNzlmMzIuYXBwcy5nb29nbGV1c2VyY29udGVudC5jb20iLCJzdWIiOiIxMDQ4NTIwMDI0NDQ3NTQxMzYyNzEiLCJlbWFpbCI6ImFub24uYXV0aG9yLmFhcmR2YXJrQGdtYWlsLmNvbSIsImVtYWlsX3ZlcmlmaWVkIjp0cnVlLCJhdF9oYXNoIjoiVmFGaGtlTE9ITXBxVWQ0RU9ZdW84ZyIsIm5vbmNlIjoiNndXMTExY25BajBlZUxzUGFDOGc5WlVkOXRDS2o1ZGNNZkt6OUZYZUFzYyIsIm5hbWUiOiJBbm9ueW1vdXMgQXV0aG9yIiwicGljdHVyZSI6Imh0dHBzOi8vbGgzLmdvb2dsZXVzZXJjb250ZW50LmNvbS9hL0FBY0hUdGRWR0Zab19aXzNoajY2ZFgzWjBHVklVUktLb2dCcGlKaDduLVhnPXM5Ni1jIiwiZ2l2ZW5fbmFtZSI6IkFub255bW91cyIsImZhbWlseV9uYW1lIjoiQXV0aG9yIiwibG9jYWxlIjoiZW4iLCJpYXQiOjE2ODUzOTU0MDEsImV4cCI6MTY4NTM5OTAwMX0",
+		"signatures": []any{
+			map[string]any{
```

Comment:
One advantage of the compact format was that if something changed and need to update a test, I could just print the compact to console and copy paste it into the test. This is more readable, how easy it is print to console? Is there a go json pretty print that will include these `JSON map[string]any{`?

Reply:
nope, i used chatgpt to do this for me because it was annoying. My hope was that people would be able to copy this object and paste in their own values or we would have a programatic way to generate these mock tokens, hopefully the latter.

Reply:
can you use the json format you used in catest.go

Reply:
As long as they are consistent, I am good with this.